dialogue/Tutorial/TutJump.json

```diff
@@ -13,7 +13,7 @@
     ],
     "pl": [
       "Tym szybczej biegasz, tym wyżej możesz skakać.",
-      "Przetrzymanie przecisk skoku jeszcze maksymalizuje Twóją wysokość skoku.",
+      "Przetrzymanie przecisk skoku jeszcze maksymalizuje Twoją wysokość skoku.",
```

Comment:
"Dłuższe przytrzymanie przycisku skoku także maksymalizuje wysokość Twojego skoku." I feel like the word "Dłuższe" is essential here as without it, it might not be clear that the longer you hold jump button the higher you jump